src/attack.js

```diff
@@ -56,16 +56,23 @@ export function getAttackUrl(text) {
  * @param {Array} techniques - Array of techniques to
  */
 export function buildAttackLayer(attackDomain, layerTitle, defaultColor,
-    techniques) {
+    techniques, colorFlag) {
     const layer = Object.assign({}, layerTemplate);
```

Comment:
While reviewing this, I found another (pre-existing) bug. This `Object.assign()` line is supposed to copy the layerTemplate, but it's a shallow copy and not a deep copy. So when we append to the techniques, we're actually appending to whatever techniques we previously exported.

Since you're working on this code already, can you fix this? I can think of two quick solutions, one is to add a line after `Object.assign` that reinitializes techniques: `layer.techniques = []` -- hacky but gets the job done. 

The other quick solution is replace the `layerTemplate` global with a function that returns a layer template -- that way a new object is created each time. A little more work but less hacky.

Reply:
I changed it to a function that returns the new layerTemplate in the same format as before, all of my testing shows that it solved the problem and only the shown bookmarks are exported.